src/main/java/flab/gotable/service/MemberService.java

```diff
@@ -25,4 +28,24 @@ public void signUp(MemberSignUpRequestDto memberSignUpRequestDto) {
     public boolean isDuplicatedId(String id) {
         return memberMapper.existId(id);
     }
+
+    @Transactional
+    public boolean findMemberByIdAndPassword(MemberLoginRequestDto memberLoginRequestDto) {
+        Member member = findMemberById(memberLoginRequestDto.getId());
+
+        if(passwordEncoder.matches(memberLoginRequestDto.getPassword(), member.getPassword())) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
이런 경우 간단하게 아래 표현으로 간결하게 작성할 수 있습니다.
```suggestion
        return passwordEncoder.matches(memberLoginRequestDto.getPassword(), member.getPassword())
```